packages/modal/src/presenters/ModalHeaderPresenter.test.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { generateId } from "@hig/utils";
+import ModalHeaderPresenter from "./ModalHeaderPresenter";
+
+describe("checkbox/presenters/ModalHeaderPresenter", () => {
+  afterEach(() => {
+    generateId.mockReset();
+  });
+
+  [
```

Comment:
_Nitpick:_ I added the `takeSnapshotsOf` test helper to reduce the amount of duplicate code.

```js
import { takeSnapshotsOf }  from '@hig/jest-preset/helpers';

takeSnapshotsOf(ModalHeaderPresenter, [ { desc: '', props: {} } ]);
```

Reply:
Nice! GTK Updated in aad2e914